doc/articles/guides/localization-and-culture-hot-swapping.md

```diff
@@ -0,0 +1,186 @@
+
+            <StackPanel toolkit:VisibleBoundsPadding.PaddingMask="Top">
+                <TextBlock x:Uid="MainPage_IntroText" Text="Hello, world!" Margin="20" FontSize="30" />
+                <TextBlock x:Name="CodeBehindText" Text="This text will be replaced" />
+            </StackPanel>
+        </Page>
+        ```
+        > note:
+        > - The [`x:Name`](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/x-name-attribute) is used to make the element accessible from the code-behind with that same name.
+        > - The [`x:Uid`](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/x-uid-directive) is used for localization.
+        To localize a property, you need to add a string resource in each resource file using its `x:Uid` followed by a dot (`.`) and then the property name. eg: `MainPage_IntroText.Text`
+        More on this in the resource steps that follow.
+    - In `MainPage.xaml.cs`, add an `Page.Loaded` handler to change the text for `CodeBehindText`:
+        ```cs
+        public MainPage()
+        {
+            this.InitializeComponent();
+            this.Loaded += (s, e) => CodeBehindText.Text = Windows.ApplicationModel.Resources.ResourceLoader
+                .GetForViewIndependentUse()
+                .GetString("MainPage_CodeBehindString");
+        }
+        ```
+1. Create a new resource file for localization in french in the `UnoLocalization.Shared` project:
+    1. Add a new folder `fr` under the `Strings` folder by:
+    R-Click on `String` > Add > New Folder
+    1. Add a new resource file `Resources.resw` under the `fr` folder by:
+    R-Click on `fr` > Add > New Item ... > Visual C# > Xaml > Resources File
+1. Add some localization texts for both english and french:
+
+    Open both `Strings\en\Resources.resw` and `Strings\fr\Resources.xaml`, and add these:
+    Name|Value in en\Resources.resw|Value in fr\Resources.resw
+    -|-|-
+    MainPage_IntroText.Text|Hello Uno|Bonjour Uno
+    MainPage_CodeBehindString|String from code-behind|Texte provenant du code-behind
+    > note: Be sure to hit Ctrl+S for both files, to save the changes.
+1. You can now try to run the app.
+
+    The "Hello World" text should be replaced with "Hello Uno", or "Bonjour Uno" if the targeted environment is on french culture.
+    Now, if you change the language of the targeted PC or the mobile device AND restart the app, that text should also change accordingly.
+
+    You can also set the starting culture without affecting the OS settings using this method:
+    - `App.xaml.cs`:
+        ```cs
+        public App()
+        {
+            ConfigureFilters(global::Uno.Extensions.LogExtensionPoint.AmbientLoggerFactory);
+            ChangeStartingLanguage();
+
+            this.InitializeComponent();
+            this.Suspending += OnSuspending;
+        }
+
+        private void ChangeStartingLanguage()
+        {
+            var culture = new System.Globalization.CultureInfo("fr");
+
+        #if WINDOWS_UWP
```

Comment:
per the discussion on the other PR, I think you can safely use this Uno 3.6 feature